test/express-error.js

```diff
@@ -12,7 +12,7 @@ var root = __dirname + '/public',
 
 mkdirp.sync(root + '/emptyDir');
 
-var cases = require('./common-cases-error');
+var cases = require('./secret/common-cases-error');
```

Comment:
[apparently we were using it wrong](https://github.com/isaacs/node-tap/issues/128#issuecomment-101453724). tap does not expect any files that are not tap test files. So I moved the non-test files into secret/ and use the following glob pattern: **test/*.js**